static/js/pages/applications/PaymentHistoryPage.js

```diff
@@ -0,0 +1,158 @@
+                  <div className="col-3 key">Payment Date:</div>
+                  <div className="col-9">
+                    {formatReadableDateFromStr(order.updated_on)}
+                  </div>
+                </div>
+                <div className="amount-paid row">
+                  <div className="col-3 key">Amount Paid:</div>
+                  <div className="col-9">
+                    {formatPrice(order.total_price_paid)}
+                  </div>
+                </div>
+                <div className="balance row">
+                  <div className="col-3 key">Balance:</div>
+                  <div className="col-9">{formatPrice(balance)}</div>
+                </div>
+                <div className="payment-method row">
+                  <div className="col-3 key">Payment Method:</div>
+                  <div className="col-9">{order.payment_method}</div>
+                </div>
+              </div>
+              <hr />
+            </div>
+          ))}
+          <div className="summary">
+            <div className="row total-paid">
+              <div className="col-3 summary-key">Total Amount Paid:</div>
+              <div className="col-9">{formatPrice(totalPaid)}</div>
+            </div>
+            <div className="row balance">
+              <div className="col-3 summary-key">Balance Due:</div>
+              <div className="col-9">{formatPrice(balanceRemaining)}</div>
+            </div>
+            <div className="row amount-due">
+              <div className="col-3 summary-key">Total Amount Due:</div>
+              <div className="col-9">{formatPrice(totalPrice)}</div>
+            </div>
+          </div>
+        </div>
+        <div className="customer inner-container">
+          <h2 className="col-12">Customer Information</h2>
+          <div className="row name">
+            <div className="col-3 key">Name:</div>
+            <div className="col-9">
+              {pathOr("", ["user", "profile", "name"], application)}
+            </div>
+          </div>
+          <div className="row email">
+            <div className="col-3 key">Email address:</div>
+            <div className="col-9">{application.user.email}</div>
+          </div>
+          <div className="row address">
+            <div className="col-3 key">Address:</div>
+            <div className="col-9">
+              <Address application={application} countries={countries} />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const mapStateToProps = (state, ownProps: OwnProps) => {
+  const applicationId = ownProps.match.params.applicationId
+  const application = path(
+    ["entities", "applicationDetail", applicationId],
+    state
+  )
+
+  return {
+    application,
+    countries: queries.users.countriesSelector(state)
+  }
+}
+const mapPropsToConfigs = (props: Props) => [
+  queries.applications.applicationDetailQuery(props.match.params.applicationId),
+  queries.users.countriesQuery()
```

Comment:
Assuming the user associated with the application is always going to be the user looking at this  payment history page, it should be possible to get the user info via ` queries.users.currentUserQuery` / `currentUserSelector` here  - or add a new `userQuery` and `userSelector` if the user might not be the same - instead of adding the user info to the `BootcampApplicationDetailSerializer`.  But either way works so whatever you think is better is fine with me.

Reply:
I think it might be useful to have the user as part of the application query in case the reviewer needs information about the applicant, though I haven't looked at the reviewer mockups yet